phpunit.xml.dist

```diff
@@ -27,6 +27,10 @@
         <!-- Enable this if you have installed ZendService\ReCaptcha on the
              include_path or via Composer. -->
         <env name="TESTS_ZEND_CAPTCHA_RECAPTCHA_SUPPORT" value="false" />
+        <!-- Change these if you want to use your own keys -->
```

Comment:
They are the official test ones. See https://developers.google.com/recaptcha/docs/faq#id-like-to-run-automated-tests-with-recaptcha-v2-what-should-i-do